lane.py

```diff
@@ -1394,7 +1373,7 @@ def groups(self, _db, include_sublanes=True, facets=None,
         if not include_sublanes:
             # We only need to find featured works for this lane,
             # not this lane plus its sublanes.
-            for work in self.featured_works(_db, facets=facets):
+            for work in self.works(_db, facets=facets):
```

Comment:
The work I'm doing in circulation is making me stop using `featured_works` earlier than I thought. I'm calling `groups` on a bunch of lanes, some of which get their data from the database and some of which get their data from the search index. `works()` is designed to handle either case, but `featured_works`, which is deprecated, can only handle the database case.